### PR TITLE
Themes | Business Dashboard - add EmptyState theme

### PR DESCRIPTION
**Pull request by just-jeb**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6262

File: src/EmptyState/test/EmptyState.visual.js [hunk 1]
```diff
@@ -143,24 +143,34 @@ const rtlTests = [
...
-    ));
+    rtlTests.forEach(({ describe, its }) => {
+      its.forEach(({ it, props }) => {
+        story(describe, () => {
+          snap(
+            it,
+            testWithTheme(
+              <RTLWrapper rtl>
+                <EmptyState {...commonProps} {...props} />
+                ))
```
Comment (shlomitc) [inline src/EmptyState/test/EmptyState.visual.js:166]:
```suggestion

```

File: src/EmptyState/EmptyState.st.css [hunk 2]
```diff
@@ -66,6 +66,12 @@
   margin-bottom: 6px;
 }
 
+.title {
+}
+
+.subtitle {
```
Comment (shlomitc) [inline src/EmptyState/EmptyState.st.css:72]: `-st-extends: Text`

File: src/EmptyState/EmptyState.js [hunk 3]
```diff
@@ -57,21 +57,32 @@ const EmptyState = ({
           {theme === 'section' ? (
             <FontUpgradeContext.Consumer>
               {({ active }) => (
-                <Text weight={active ? 'bold' : 'normal'}>{title}</Text>
+                <Text
+                  weight={active ? 'bold' : 'normal'}
+                  className={classes.title}
```
Comment (shlomitc) [inline src/EmptyState/EmptyState.js:62]: `classes.sectionTitle` and `-st-extends: Text`

File: src/EmptyState/EmptyState.js [hunk 4]
```diff
@@ -57,21 +57,32 @@ const EmptyState = ({
...
+                >
+                  {title}
+                </Text>
               )}
             </FontUpgradeContext.Consumer>
           ) : (
-            <Heading appearance="H3">{title}</Heading>
+            <Heading className={classes.title} appearance="H3">
```
Comment (shlomitc) [inline src/EmptyState/EmptyState.js:69]: `-st-extends: Heading`

**Review by shlomitc — CHANGES_REQUESTED** [review]
minor changes

**Pull request merged**